### PR TITLE
Fixed several products

### PR DESCRIPTION
**Pull request by doamatto**
**Type of pull request:** Product edit(s)

---

**Changes**
- Added self as contributor for products mentioned below
- Fixed behavioural marketing citations for 2ndLine
- Fixed behavioural marketing citations and last modified for AliExpress
- Fixed name for clarity and behavioural marketing citations for Amazon US
- Fixed law enforcement, ~~data breach~~, and collecting data from third-parties for BBC
- Fixed data deletion for Calendly
- Fixed data deletion and data security for Discord
- Added hostnames for other Discord domains
- Fixed privacy policy source URL for Discord
- Fixed behavioural marketing for Verizon
- Fixed behavioural marketing for Visible
- Added Shazam, iCloud, and Dark Sky, and nested them to Apple's policy
- Removed duplicates from Apple policy

### CODE REVIEW

**Comment by doamatto** [comment]
I could be wrong, but the CI seems bugged. It's saying there isn't a Shazam icon, but there is a vector  [here](https://github.com/doamatto/privacyspy/blob/patch-1/icons/shazam.svg) and removed the past 2 errors of it.

**Comment by milesmcc** [comment]
It’s not bugged. The slug for Shazam must exactly match the name of the icon. Change Shazam’s slug to be lowercase to fix the issue (slugs should always be lowercase; we should add a test for this).

**Comment by milesmcc** [comment]
Thanks for these fixes!

**Comment by doamatto** [comment]
> It’s not bugged. The slug for Shazam must exactly match the name of the icon. Change Shazam’s slug to be lowercase to fix the issue (slugs should always be lowercase; we should add a test for this).
@milesmcc It does match the slug; it's apparently because it isn't mostly square. I must have missed it in some documentation somewhere. I fixed the upload; should be ready for merge now.

**Comment by milesmcc** [comment]
Ah good catch. I'm so used to people mixing up the capitalization that I missed the 'mostly square' part!

File: products/bbc.toml [hunk 1]
```diff
@@ -10,8 +10,8 @@ value = "yes-opt-in"
 citations = ["We'll only send you marketing emails or contact you about BBC programmes, services and your views on issues about the BBC if you've agreed to this."]
 
 [rubric.data-breaches]
-value = "no"
-notes = ["This policy does not say what will happen in the event of a data breach."]
+value = "na"
```
Comment (ibarakaiev) [inline products/bbc.toml:13]: I don't really understand why it makes sense to have `na` here. Can you clarify? Naturally, if a user doesn't provide an email then they shouldn't be notified, but it seems like there is an option to create an account so `na` is not applicable.

File: products/bbc.toml [hunk 2]
```diff
@@ -73,8 +73,8 @@ g. Sometimes by law we have to pass on your information to other organisations
 We might also share your information if we have to by law, or when we need to protect you or other people from harm."""]
 
 [rubric.third-party-collection]
-value = "yes"
-citations = ["Bear in mind there are some other cookies out there from other companies. These 'third-party cookies' might track how you use different websites, including ours. For example, you might get a social media company’s cookie when you see the option to share something. You can turn them off, but not through us."]
+value = "no"
+notes = "There is no mention of collecting data from third parties."
```
Comment (ibarakaiev) [inline products/bbc.toml:77]: Good catch that they don't collect data from third parties. However, `notes` should be an array.

**Review by ibarakaiev — CHANGES_REQUESTED** [review]
Left two comments on BBC; everything else looks good.

File: products/bbc.toml (hunk 1 above)
Comment (doamatto) [reply products/bbc.toml:13]: In that case, I'll revert these changes.

File: products/bbc.toml (hunk 2 above)
Comment (doamatto) [reply products/bbc.toml]: My bad; fixed

**Comment by doamatto** [comment]
> Left two comments on BBC; everything else looks good.

Alright; I've fixed those issues and re-requested a review.

**Comment by ibarakaiev** [comment]
Looks good, thanks :)

**Pull request merged**